context/AuthProvider.js

```diff
@@ -16,17 +17,35 @@ const AuthProvider = (props) => {
   const [user, setUser] = useState({});
 
   const signIn = (params) => {
-    console.log(params, 'sign in form Props');
-    setUser(fakeUserData);
-    setLoggedIn(true);
-    Router.push(`/`);
+    console.log(params, 'sign in form Props'); 
+        setUser(fakeUserData);
```

Comment:
Identation

Reply:
Remove this console.log please

Reply:
this will be changed when you work on the sign in. Do I fix it here anyway?

Reply:
No, no problem.